package.json

```diff
@@ -74,6 +74,7 @@
     "megalog": "^0.0.2",
     "mkdirp": "0.5.0",
     "moment": "2.8.3",
+    "shoe": "0.0.11",
```

Comment:
See the related issue in the commit https://github.com/guardian/frontend/commit/6a9ef7cabdd9823f4937629bde9003c10cc4f082